src/p2pool/P2PoolManager.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2014-2019, The Monero Project
```

Comment:
```suggestion
// Copyright (c) 2014-2022, The Monero Project
```

Same for the other new files.